src/main/java/ru/hogwarts/school/model/Faculty.java

```diff
@@ -0,0 +1,65 @@
+package ru.hogwarts.school.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import nonapi.io.github.classgraph.json.Id;
```

Comment:
```suggestion
import jakarta.persistence.Id;
```
Нужно поправить импорт